src/app/components/core/notifications/notifications.component.ts

```diff
@@ -39,6 +41,38 @@ export class NotificationsComponent implements OnInit {
     public router: Router) { }
 
   ngOnInit() {
+    this.getAvailableServices();
+    this.getNotificationInstance();
+  }
+
+  public getAvailableServices() {
+    /** request start */
+    this.ngProgress.start();
+    this.servicesApiService.getAvailableServices()
+      .subscribe(
+        (data: any) => {
+          /** request done */
+          this.ngProgress.done();
+          this.availableServices = data.service;
+          if (this.availableServices.length >= 1) {
```

Comment:
Also please ensure it is foglamp-service-notification, in case there are more services packages in future this will cause problems